.pipelines/cni/scale-test-templates/apache.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+   name: apachebench
+spec:
+  selector:
+    matchLabels:
+      app: apachebench
+  template:
+    metadata:
+      labels:
+        app: apachebench
+    spec:
+      containers:
+      - name: ubuntu-tools
+        image: tamilmani1989/ubuntu18-tools
```

Comment:
Can we see if we can a substitute for this image ?
Also @tamilmani1989 should we consider ubuntu 22 image in that case ?

Reply:
Last week we confirmed that we can continue to use this image

Reply:
this is fine.. this is just pod image